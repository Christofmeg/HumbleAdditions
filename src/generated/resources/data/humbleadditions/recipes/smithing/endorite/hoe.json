{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorite_block"
  },
  "base": {
    "item": "humbleadditions:endorium_hoe"
  },
  "result": {
    "item": "humbleadditions:endorite_hoe"
  }
}