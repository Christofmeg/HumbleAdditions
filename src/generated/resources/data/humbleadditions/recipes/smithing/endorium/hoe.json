{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorium_block"
  },
  "base": {
    "item": "minecraft:diamond_hoe"
  },
  "result": {
    "item": "humbleadditions:endorium_hoe"
  }
}