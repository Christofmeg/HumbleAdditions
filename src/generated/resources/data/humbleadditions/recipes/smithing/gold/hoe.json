{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_block"
  },
  "base": {
    "item": "minecraft:stone_hoe"
  },
  "result": {
    "item": "minecraft:golden_hoe"
  }
}