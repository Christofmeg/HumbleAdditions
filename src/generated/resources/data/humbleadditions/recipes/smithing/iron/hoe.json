{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_block"
  },
  "base": {
    "item": "minecraft:golden_hoe"
  },
  "result": {
    "item": "minecraft:iron_hoe"
  }
}