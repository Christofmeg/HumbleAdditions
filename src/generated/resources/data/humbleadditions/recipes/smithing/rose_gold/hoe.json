{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:rose_gold_ingot"
  },
  "base": {
    "item": "minecraft:golden_hoe"
  },
  "result": {
    "item": "humbleadditions:rose_gold_hoe"
  }
}