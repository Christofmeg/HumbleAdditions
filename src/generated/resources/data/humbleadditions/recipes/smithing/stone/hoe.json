{
  "type": "minecraft:smithing",
  "addition": {
    "tag": "minecraft:stone_tool_materials"
  },
  "base": {
    "item": "minecraft:wooden_hoe"
  },
  "result": {
    "item": "minecraft:stone_hoe"
  }
}