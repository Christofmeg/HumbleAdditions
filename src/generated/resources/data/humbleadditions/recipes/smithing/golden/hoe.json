{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_ingot"
  },
  "base": {
    "item": "minecraft:stone_hoe"
  },
  "result": {
    "item": "minecraft:golden_hoe"
  }
}